### DISCUSSION TITLE
Geospatial data in MySQL

### DISCUSSION THREAD

**Discussion by SirCameron** [prisma/prisma#4946] [Q&A]
Hi

Does Prisma support MySQL geospatial data and column types?
I've been unable to find anything about this in the docs.

**Comment by ryands17** [comment]
Hey @SirCameron 👋 

Currently geospatial data and column types aren't directly supported, but you can use [Introspection](https://www.prisma.io/docs/concepts/components/introspection) and [raw queries](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) as a workaround. We have an example [here](https://github.com/prisma/prisma-examples/tree/latest/typescript/postgis-express) that demonstrates it.

We also have a [feature request](https://github.com/prisma/prisma/issues/1798) for the same so do add a 👍 so that we can know the priority for this :)

**Reply by SirCameron** [reply]
Thanks @ryands17 !

**Comment by muhaimincs** [comment]
in 2024, how's the work around for this? Particularly MySQL

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
